net/keepalived/Makefile

```diff
@@ -241,6 +242,10 @@ endif
 	$(INSTALL_DATA) ./files/hotplug-user \
 		$(1)/etc/hotplug.d/keepalived/01-user
 
+	$(INSTALL_DIR) $(1)/usr/libexec/rpcd
```

Comment:
Since the output depends on the compile option. So we should not always install the rpcd?
But I am generally in favour of always switching it on and removing the compile option.
Especially since we need the ubus output in the LuCI, right?

Reply:
Yes, Enabled JSON compilation in the default builds and removed the option.